### PR TITLE
Template updates

### PR DESCRIPTION
**Pull request by ricellis**
Thanks for your hard work, please ensure all items are complete before opening.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://github.com/cloudant/java-cloudant/blob/master/DCO1.1.txt)
- [x] You have added tests for any code changes - N/A
- [x] You have updated the [CHANGES.md](https://github.com/cloudant-labs/cloudant-spring/blob/master/CHANGES.md) - N/A
- [x] You have completed the PR template below:

## What

Updated issue and PR templates and contributing file.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+<!--
+Got a 400 bad request
+-->
+
+## Environment details
+<!--
+- Version(s) that are affected by this issue.
+    > 2.0.2
+- Java version including vendor and platform
+    >1.8.0_151 OpenJDK macOS
```
Comment (emlaver) [inline .github/ISSUE_TEMPLATE.md]: Add space

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- Spring framework or boot version
+    >Spring boot 2.0.2
```
Comment (emlaver) [inline .github/ISSUE_TEMPLATE.md]: Add space

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,18 +26,33 @@ Please read the agreement and acknowledge it by ticking the appropriate box in t
 
 - [x] Tick to sign-off your agreement to the Developer Certificate of Origin (DCO) 1.1
 
+<!-- Append library specific information here
```
Comment (emlaver) [inline CONTRIBUTING.md]: Remove this since everything below is commented and hidden in markdown view.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ricellis) [reply .github/ISSUE_TEMPLATE.md]: Fixed in 7eccd2ce901c244b067a94499027636605eb1a3a

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ricellis) [reply .github/ISSUE_TEMPLATE.md]: Fixed in 7eccd2ce901c244b067a94499027636605eb1a3a

File: CONTRIBUTING.md (hunk 3 above)
Comment (ricellis) [reply CONTRIBUTING.md]: Fixed in 0f19672b217943ef931b6274e724df1612662c7b

**Pull request merged**